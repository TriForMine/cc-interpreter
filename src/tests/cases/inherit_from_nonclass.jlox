var a = "this is not a class";

class Bagel < a { }