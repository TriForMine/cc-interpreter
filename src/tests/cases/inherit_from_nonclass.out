Error: Superclass must be a class, not a string